main.go

```diff
@@ -0,0 +1,239 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+
+	"encoding/json"
+
+	"github.com/Shopify/sarama"
+	"github.com/companieshouse/chs.go/avro"
+	"github.com/companieshouse/chs.go/avro/schema"
+	"github.com/companieshouse/chs.go/kafka/producer"
+	"github.com/companieshouse/kafka-management-tool/schemas"
+)
+
+// Assigns all flags to variables
+var (
```

Comment:
any reason we didn't use gofigure for these flags?
https://github.com/companieshouse/gofigure

would make it easier to set up specific jobs allowing the use of environment variables and/or command line flags

Reply:
I'm not sure if we really need it. Importing an extra library for something that can be done via the normal flags library. Also I don't know when a user would ever use environment variables to run this as its a command line tool

Reply:
probably just a taste thing and a nice to have so wouldn't stop the PR going in but for the more environmental options (broker address and schema registry address) it'd be nice to be able to have those sourced in as env vars to make the commands easier to run/remember between environments

Reply:
Ahh I see, in that case, it indeed would be a nice thing to add. Is it worth creating an issue for it which I can pick up later on? Just so we can get the barebones of the tool into develop so support can start testing it.